.travis.yml

```diff
@@ -17,7 +17,7 @@ before_install:
 script:
   - source test/env.sh
   - ./rebar3 compile
```

Comment:
traivs has rebar3 support builtin now, so the wget for it and the relative path invocation can be dropped. Simply using ```rebar3 ...`` should work

Reply:
Should I do this in the PR? Maybe we can separate it to no mix redirector with `rebar3`...

Reply:
I've removed it and switched dist to xenial in #64